cpp/data/system_updates/123.vufind.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tuefind_publications ADD doi_link VARCHAR(255) NULL AFTER external_document_guid;
```

Comment:
Do we really need to store the whole URL in the database? I think the DOI itself would be enough

Reply:
As discussed, we should still have an additional column called `doi`

Reply:
Looks better now! Don't forget to open a PR in ubtue/tuefind as well, which adds these columns to `module/TueFind/sql/mysql.sql`